src/hyperlink/_url.py

```diff
@@ -815,9 +815,9 @@ class URL(object):
           that starts with a slash.
        userinfo (Text): The username or colon-separated
           username:password pair.
-       uses_netloc (bool): Indicates whether two slashes appear
-          between the scheme and the host (``http://eg.com`` vs
-          ``mailto:e@g.com``). Set automatically based on scheme.
+       uses_netloc (bool): Indicates whether ``://`` will appear to separate
+           the scheme from the path, even in cases where no host is present.
+           May be implied by scheme, or set explictly.
```

Comment:
Should this say that one probably should only set it explicitly if you've registered a scheme not known to hyperlink?